packages/elements/src/internals/machines/sign-in/router.machine.ts

```diff
@@ -426,7 +410,11 @@ export const SignInRouterMachine = setup({
     },
     'FORM.ATTACH': {
       description: 'Attach/re-attach the form to the router.',
-      actions: enqueueActions(({ enqueue, event }) => {
+      actions: enqueueActions(({ context, enqueue, event }) => {
```

Comment:
Ensuring we don't attach a new formRef, and ultimately re-render, if it's not necessary.